internal/infra/updater.go

```diff
@@ -53,6 +53,11 @@ func NewUpdater(ctx context.Context, cli *client.Client, net *Networks, params *
 		Cmd:   []string{"/bin/sh"},
 		Tty:   true, // prevent container from stopping
 	}
+
+	if params.EnableOpenTelemetry == true {
```

Comment:
q: if `params.EnableOpenTelemetry` is `false` do we need to explicitly set the `containerCfg.Env` `OTEL_ENABLED` to false?

Reply:
No, undefined is accepted as `false` too.